### ISSUE TITLE
SetCompatibilityVersion Setting Wrong Version

### ISSUE THREAD

**Issue by TheFanatr** [aspnet/AspNetCore.Docs#12446]
Shouldn't the first code block on this page use `SetCompatibilityVersion` to set the compatibility version to `CompatibilityVersion.Version_3_0` considering that this page is present in the ASP.NET Core 3.0 Preview view in the documentation?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f3a49eaa-a2ee-baac-e614-5e2f7c74072a
* Version Independent ID: 09513dc8-546a-ea10-9f0b-cde361127647
* Content: [Razor Pages authorization conventions in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/security/authorization/razor-pages-authorization?view=aspnetcore-3.0#feedback)
* Content Source: [aspnetcore/security/authorization/razor-pages-authorization.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/security/authorization/razor-pages-authorization.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-security**
* GitHub Login: @guardrex
* Microsoft Alias: **riande**

**Comment by guardrex** [comment]
Hello @TheFanatr ... AFAIK, we have a technical limitation there. For versioning the whole topic, we use a single conditional (e.g., `>=`) and identifier (`aspnetcore-X.Y`, where X is major and Y is minor). For example, this topic uses ...

```
monikerRange: '>= aspnetcore-2.1'
```

... so any version selected at or above 2.1 loads the topic normally (e.g., 2.0 throws up that 'this topic isn't available for the version selected' notice, but 3.0 Preview won't).

We have a way in code blocks to provide multiple version conditionals. For example ...

```
::: moniker range=">= aspnetcore-2.0 <= aspnetcore-2.2"

...

::: moniker-end
```

@scottaddie I have a sneaky feel'in the following options don't exist today:

* `monikerRange: '>= aspnetcore-2.1 < aspnetcore-3.0'`
* `monikerRange: '= aspnetcore-2.1 = aspnetcore-2.2'`

**Comment by guardrex** [comment]
This issue will be addressed shortly on the 3.0 topic update sweep. Closing here and will track the update with the rest of the 3.0 updates.
